pep/app/pods/components/document/text/component.ts

```diff
@@ -18,9 +20,59 @@ export default class DocumentText extends Component<DocumentTextArgs> {
     @service loadingBar!: LoadingBarService;
     @service modal!: ModalService;
 
+    @tracked xml?: XMLDocument;
+
+    constructor(owner, args) {
+        super(owner, args);
+        const XML = parseXML(this.args.text);
```

Comment:
we should guard against running any of this in fastboot at all, so that it doesnt even generate an `Error`. maybe move all this logic to a method with the `@dontRunInFastboot` decorator, and call that from the constructor.